geomesa-jupyter/geomesa-jupyter-vegas/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>geomesa-jupyter_2.11</artifactId>
+        <groupId>org.locationtech.geomesa</groupId>
+        <version>1.3.1-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>geomesa-jupyter-vegas_2.11</artifactId>
+    <name>GeoMesa Jupyter Vegas Runtime</name>
+
+    <profiles>
+        <profile>
+            <id>vegas</id>
+
+            <dependencies>
+                <dependency>
+                    <groupId>org.vegas-viz</groupId>
+                    <artifactId>vegas-spark_2.11</artifactId>
+                    <version>0.3.6</version>
+                </dependency>
+            </dependencies>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-shade-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>shade</goal>
+                                </goals>
+                                <configuration>
+                                    <createDependencyReducedPom>false</createDependencyReducedPom>
+                                    <artifactSet>
+                                        <excludes>
+                                            <!-- provided by Spark -->
+                                            <exclude>org.scala-lang:*</exclude>
```

Comment:
Ok, so there are exclusions here for Scala.  Someone was mentioning that Vegas pulls in Spark.  

Is that the case?  Do we need other dependencies here?